view/frontend/web/js/private-data.js

```diff
@@ -0,0 +1,23 @@
+define([
+    'uiComponent',
+    'Magento_Customer/js/customer-data',
+    'jquery'
+], function (Component, customerData, $) {
+    'use strict';
+
+    return Component.extend({
+        /** @inheritdoc */
+        initialize: function () {
+            this._super();
+            this.metrilosection = customerData.get('metrilo-private-data');
+
+            if (this.metrilosection) {
+                this.metrilosection.subscribe(function () {
+                    $.each(this.metrilosection().metrilo_events, function(key, val){
+                        eval(val.replace("window.", ""));
```

Comment:
Why do you have to remove `window`?

Reply:
Hmm, this is mistake and should not be removed. Will fix it.